plugins/aiurdemux/aiurstreamcache.c

```diff
@@ -33,6 +33,7 @@
  * Changelog:
  *
  */
+ #include <gst/gstinfo.h>
```

Comment:
here is the error
```
../git/plugins/aiurdemux/aiurstreamcache.c:37:28: error: a parameter list without types is only allowed in a function definition
GST_DEBUG_CATEGORY_EXTERN (aiurdemux_debug);
                           ^
```

Reply:
seems, this error is only for Clang? I cannot see this error when using gcc

Reply:
newer clang16+ does error out on  types that do not have prototypes but older compilers just warned about it. Thats why you see error now since its C99 requirement